cmd/export_release.go

```diff
@@ -41,3 +47,34 @@ func (c ExportReleaseCmd) Run(opts ExportReleaseOpts) error {
 
 	return nil
 }
+func (c ExportReleaseCmd) ensureJobsValidForRelease(jobs []string, rel boshdir.ReleaseSlug) error {
+	releases, err := c.deployment.Releases()
+	if err != nil {
+		return err
+	}
+	releaseJobs := make([]string, 0)
+	for _, release := range releases {
+		if release.Name() == rel.Name() &&
+			release.Version().String() == rel.Version() {
+			jobs, err := release.Jobs()
+			if err != nil {
+				return err
+			}
+			for _, job := range jobs {
+				releaseJobs = append(releaseJobs, job.Name)
+			}
+		}
+	}
+	for _, job := range jobs {
+		if !contains(releaseJobs, job) {
+			return fmt.Errorf(
+				"Job '%s' for release '%s' doesn't exist", job, rel)
+		}
+	}
+	return nil
+}
+
+func contains(s []string, searchterm string) bool {
+	i := sort.SearchStrings(s, searchterm)
```

Comment:
Is the job list from the director already sorted by name?

Reply:
lets not assume that, will add a test and fix it